pandas/core/frame.py

```diff
@@ -2504,9 +2504,14 @@ def transpose(self, *args, **kwargs):
 
         Parameters
         ----------
-        *args, **kwargs
-            Additional arguments and keywords have no effect but might be
-            accepted for compatibility with numpy.
+        *args : tuple, optional
+            Accepted for compatibility with NumPy.
+        copy : bool, default False
+            Whether to copy the data after transposing, even for DataFrames
```

Comment:
versionadded tag

Reply:
FYI, copy isn't new. It's been around for a long time I think (the docstring in NDFrame.transpose didn't have a versionadded).